docs/solution-design.md

```diff
@@ -6,6 +6,50 @@ While there are many ways to achieve the goal we are looking for, for the purpos
 
 ## Components used and their roles
 
-## Message Flow
+The following components will be used as part of the solution:
+  
+- Cosmos DB Account
+- Cosmos DB Containers
+  - On Hand (onhand)
+  - Shipments (shipments)
+  - Master Data Container (mdc)
+- Azure Functions
+  - Change Feed Listeners
+  - Durable Function Orchestrator
+  - Durable Entity as a Virtual Actor
+
+As part of the challenges, we will deploy the Cosmos DB Account and Containers on Azure and build the Azure Functions to process the changes.
+
+## Processing Sequence
+
+The following diagram describes the interaction between the various components:
+
+![Processing Flow](./images/OpenHack_ProcessingFlow.png)
+
+The processing starts when documents are fed in to the Cosmos DB containers (shipments and on hand), which triggers a change feed in CosmosDB.
+
+Azure Functions configured to listen to the feed get a list of documents which are changed (add/edited/deleted). The Change feed listener convert them to events.
```

Comment:
Change feed doesn't respond to document deletes, if we want to add this we will have to add a "soft delete" property and use the ttl to expire the documents

Reply:
I will limit to the case we are using it for.